client/src/modules/login/login.html

```diff
@@ -132,6 +132,9 @@ <h1 translate class="text-center">INSTALL.TITLE</h1>
             </form>
           </div>
         </div>
+        <div class="helpdesk-info" ng-if="LoginCtrl.helpdesk_info">
```

Comment:
Let's put this in a `class="alert alert-info"` box to draw attention to it.  Doing so, makes it look like this:

![image](https://user-images.githubusercontent.com/896472/97958724-1643f700-1dae-11eb-8893-5168cbb3b311.png)

Reply:
This is okay for now.  However, when we actually start putting information here, the blue of the 'alert' area may not fit in with the visualization that we want.  We will probably need to develop custom CSS rules for this area--which we can do later.